src/modules/multilevelFacets/generateFacets/GenerateFacets.js

```diff
@@ -119,6 +119,9 @@ class GenerateFacets extends React.Component {
             };
         });
     };
+    static defaultProps = {
```

Comment:
Can you please add this on the main component for consistency. In this case it would be `MultilevelFacets .js`. This makes sure that all defaults are set on the main component. Even in case of debugging we know where to look.  [ref](https://github.com/unbxd/react-search-JS-SDK/blob/045075d61b1bd800602fbb8189ee0892236e9346/src/modules/pagination/Pagination.js)

Reply:
done